docs/console/docs/022-cr-clustermicrofrontend.md

```diff
@@ -0,0 +1,50 @@
+---
+title: ClusterMicroFrontend
+type: Custom Resource
+---
+
+The `clustermicrofrontend.ui.kyma-project.io` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows to extend the Console for the entire Cluster. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd clustermicrofrontends.ui.kyma-project.io -o yaml
+```
+
+## Sample Custom Resource
+
+This is a sample CR that extends the Console. 
+
+```
+apiVersion: ui.kyma-project.io/v1alpha1
+kind: ClusterMicroFrontend
+metadata:
+  name: sample-microfrontend
+spec:
+  version: 0.0.1
+  category: category-name
+  viewBaseUrl: https://sample-microfrontend-url.com
+  placement: cluster
+  navigationNodes:
+    - label: Sample List
+      navigationPath: items
+      viewUrl: /
+    - label: Details
+      navigationPath: items/:id
+      showInNavigation: false
+      viewUrl: /:id
+```
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Field   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** | **YES** | Specifies the name of the ClusterMicroFrontend. |
+| **spec.version** | **NO** | Specifies the version of the ClusterMicroFrontend. |
+| **spec.category** | **NO** | Defines the Category name under which the ClusterMicroFrontend appears in the navigation. |
```

Comment:
`Specifies the category name under which the cluster micro front-end appears in the navigation. `